cmd/gateway-gcs.go

```diff
@@ -39,11 +38,36 @@ import (
 )
 
 const (
-	// ZZZZMinioPrefix is used for metadata and multiparts. The prefix is being filtered out,
-	// hence the naming of ZZZZ (last prefix)
-	ZZZZMinioPrefix = "ZZZZ-Minio"
+	// gcsMinioMeta is used for multiparts. We have "minio.sys.temp" prefix so that
+	// listing on the GCS lists this entry in the end. Also in the gateway
+	// ListObjects we filter out this entry.
+	gcsMinioPath = "minio.sys.temp"
```

Comment:
yes, @abperiasamy said it looks odd and this is better
Also clients will filter based on the prefix name and not what appears at the end of the list